core/server/api/canary/newsletters-public.js

```diff
@@ -0,0 +1,29 @@
+const models = require('../../models');
+
+const allowedIncludes = ['count.posts', 'count.members'];
```

Comment:
Do we want to expose these? I guess probably 🤔

Reply:
🤔 I think we should probably only expose them once we have a usecase for them ? Though in that case we could also remove them from allowed includes ?

Reply:
Removed the `includes` for now till we have a usecase for them again.